.github/workflows/build-and-run-tests.yml

```diff
@@ -17,15 +17,17 @@ jobs:
       - name: Checkout code onto job runner
         uses: actions/checkout@v2
 
-      - name: Install .Net (6.0)
+      - name: Install .Net (8.0)
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: 6.0.x
+          dotnet-version: 8.0.x
 
       - name: Restore .Net dependencies
+        env: {DOTNET_NUGET_SIGNATURE_VERIFICATION: false}
         run: dotnet restore --configfile nuget.config
 
       - name: Build .Net code
+        env: {DOTNET_NUGET_SIGNATURE_VERIFICATION: false}
```

Comment:
Looks like this hasn't worked, but for what it's worth I think Linux vs Windows might be a likely culprit here!

Reply:
I'm not completely sure. I think its happening on Windows now as well, since my PC restarted. I'm wondering if its some sort of caching thing?

I'm waiting on a windows build right now, to see if it builds faster the second time.

Reply:
Hmm yes fair enough. It is a bit annoying that we're using docker but not in a way that we can replicate what's happening in CI.